dev/cognitiveservices/data-plane/Language/questionanswering-generate.json

```diff
@@ -450,47 +369,47 @@
         },
         "content": {
           "type": "string",
-          "description": "The input text to process."
+          "description": "The input text, html or URL to process."
         },
         "documentType": {
           "type": "string",
-          "$ref": "#/definitions/QnAGenDocumentType"
+          "$ref": "#/definitions/DocumentType"
         },
         "language": {
           "type": "string",
           "description": "(Optional) This is the 2 letter ISO 639-1 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default."
         }
       },
-      "description": "Contains an input document to be analyzed by the service."
+      "description": "An input document to be processed for question answer generation."
     },
-    "QnAGenDocumentType": {
+    "DocumentType": {
```

Comment:
I missed this last time, but we actually prefer "kind" to "type" as enums and discriminators, if that's okay here.